src/style.css

```
@@ -0,0 +1,15 @@
+.body {
+  margin: 0;
+}
+
+.input {
+  margin-bottom: 10px;
+}
+
+.input:last-child {
+  margin-bottom: 0;
+}
```

це можна об'єднати в один селектор
```suggestion
.input:not(:last-child) {
  margin-bottom: 10px;
}
```